tests/FromCommunity/IssueWithQueryArrayTest.php

```diff
@@ -12,62 +12,183 @@ final class IssueWithQueryArrayTest extends TestCase
 {
     public function testConvertFormIntegerArray() : void
     {
-        $validator = (new ValidatorBuilder())->fromYaml($this->makeYaml('form', 'integer', 'int32'))->getServerRequestValidator();
+        $validator = (new ValidatorBuilder())->fromYaml($this->makeArrayYaml('form', 'integer', 'int32'))->getServerRequestValidator();
         $validator->validate($this->makeRequest('form', 'integer'));
         $this->addToAssertionCount(1);
     }
 
     public function testConvertFormNumberArray() : void
     {
-        $validator = (new ValidatorBuilder())->fromYaml($this->makeYaml('form', 'number', 'float'))->getServerRequestValidator();
+        $validator = (new ValidatorBuilder())->fromYaml($this->makeArrayYaml('form', 'number', 'float'))->getServerRequestValidator();
         $validator->validate($this->makeRequest('form', 'number'));
         $this->addToAssertionCount(1);
     }
 
     public function testConvertFormIntegerArrayToStringArray() : void
     {
-        $validator = (new ValidatorBuilder())->fromYaml($this->makeYaml('form', 'string', 'int32'))->getServerRequestValidator();
+        $validator = (new ValidatorBuilder())->fromYaml($this->makeArrayYaml('form', 'string', 'int32'))->getServerRequestValidator();
         $validator->validate($this->makeRequest('form', 'integer'));
         $this->addToAssertionCount(1);
     }
 
     public function testConvertFormStringArray() : void
     {
-        $validator = (new ValidatorBuilder())->fromYaml($this->makeYaml('form', 'string', 'int32'))->getServerRequestValidator();
+        $validator = (new ValidatorBuilder())->fromYaml($this->makeArrayYaml('form', 'string', 'int32'))->getServerRequestValidator();
         $validator->validate($this->makeRequest('form', 'string'));
         $this->addToAssertionCount(1);
     }
 
     public function testConvertFormBooleanArray() : void
     {
-        $validator = (new ValidatorBuilder())->fromYaml($this->makeYaml('form', 'boolean', 'int32'))->getServerRequestValidator();
+        $validator = (new ValidatorBuilder())->fromYaml($this->makeArrayYaml('form', 'boolean', 'int32'))->getServerRequestValidator();
         $validator->validate($this->makeRequest('form', 'boolean'));
         $this->addToAssertionCount(1);
     }
 
     public function testConvertFormIntegerArrayError() : void
     {
         $this->expectExceptionMessage('Value "id1,id2,id3" for argument "id" is invalid for Request [get /users]');
-        $validator = (new ValidatorBuilder())->fromYaml($this->makeYaml('form', 'integer', 'int32'))->getServerRequestValidator();
+        $validator = (new ValidatorBuilder())->fromYaml($this->makeArrayYaml('form', 'integer', 'int32'))->getServerRequestValidator();
         $validator->validate($this->makeRequest('form', 'string'));
         $this->addToAssertionCount(1);
```

Comment:
Didn't noticed this on prev one, but the same thing is here for new tests. There is no need in this call or `error` test - this line of code is not reachable at all. Could you please clean it up?

Reply:
Removed as requested